### PR TITLE
LMB-129 Table refactor

### PR DESCRIPTION
**Pull request by bfidlers**
Three tables have been properly refactored:
- mandataris table
- functionaris table
- fractie table
They now all use the AuDataTable and have sorting and pagination. 
In the future it would be nice to combine this in one component however.

The instance table also received a refactor, but does not allow for sorting etc, this is a problem that should be looked into at a later time if needed (this needs changes in the form-content service as well).

### CODE REVIEW

File: app/components/form/instance-table.hbs [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
-        Form Instances</AuHeading>
-    </Group>
-    <Group>
-      <AuButton {{on "click" @onCreate}}>Voeg form instance toe</AuButton>
-    </Group>
-  </AuToolbar>
+<AuToolbar @size="large" class="au-u-padding-bottom-none" as |Group|>
+  <Group>
+    <AuHeading @skin="2">Beheer
+      {{this.formInfo.form.definition.id}}
```
Comment (ruben-bottu) [inline app/components/form/instance-table.hbs:4]: Shouldn't this be this.formInfo.formDefinition.id?

File: app/components/mandatenbeheer/mandataris/mandataris-table.hbs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      <th></th>
+    </c.header>
+
+    <c.body as |row|>
+      <td>
+        {{#each
+          (get @mandatarisBestuursorganen row.id)
+          as |bestuursorgaan index|
+        }}
+          {{#if (gt index 0)}}<br />{{/if}}
```
Comment (ruben-bottu) [inline app/components/mandatenbeheer/mandataris/mandataris-table.hbs:52]: This seems a bit hacky, but ok

**Review by ruben-bottu — APPROVED** [review]

**Review by Rahien — APPROVED** [review]
very pretty!

File: app/components/form/instance-table.hbs (hunk 1 above)
Comment (bfidlers) [reply app/components/form/instance-table.hbs:4]: good catch, seems like it was broken before as well.

File: app/components/mandatenbeheer/mandataris/mandataris-table.hbs (hunk 2 above)
Comment (bfidlers) [reply app/components/mandatenbeheer/mandataris/mandataris-table.hbs:52]: Not my code, but should resolve itself in the future, with the new wireframes.

**Pull request merged**